.github/workflows/release.yaml

```diff
@@ -94,7 +92,7 @@ jobs:
                 insecure = true
                 http = true
           EOF
-          earthly +build-framework-image --FLAVOR=${FLAVOR}
+          earthly +build-framework-image --FRAMEWORK_FLAVOR=${{ matrix.flavor }}
```

Comment:
not valid anymore, framework images are now either generic or fips. We should just hardcode this to generic for now and we are just publishing generic images, not fips

Reply:
but there is a matrix defined above (line 65) to "generic, fips". That should work fine no?

Reply:
true, I though I removed this already!